bcl2fastq/lib/bcl2fastq_utils.py

```diff
@@ -41,9 +45,16 @@ def __init__(self,
         """
 
         self.runfolder_input = runfolder_input
-        self.samplesheet_file = runfolder_input + "/SampleSheet.csv"
         self.base_calls_input = runfolder_input + "/Data/Intensities/BaseCalls"
 
+        if not samplesheet:
```

Comment:
Should we ignore error handling for now? E.g. if all the paths exists and are valid/readable/writeable, or if some really obvious garbage (whitespace? other sanity check?) is sent in as a samplesheet?